roles/container_runtime/tasks/extra_storage_setup.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Create file system on extra volume device
+  filesystem:
+    fstype: "{{ item.filesystem | default('xfs') }}"
+    dev: "{{ item.device }}"
+  when:
+    - item.format|default(false)|bool
+    - inventory_hostname in groups[item.hosts|default('all')]
+
+- name: Create mount entry for extra volume
+  mount:
+    path: "{{ item.path }}"
+    src: "{{ item.device }}"
+    fstype: "{{ item.filesystem | default('xfs') }}"
+    opts: "{{ item.options|default('defaults') }}"
```

Comment:
Don't define any defaults; user should specify everything.

Reply:
@michaelgugino Would this apply to a `default(omit)` as well? For file system options, it makes more sense to only touch it if we need to add one, (like `grpquota`) otherwise use the defaults.